code/modules/ghostroles/spawner/human/human.dm

```diff
@@ -37,9 +37,12 @@
 		if(mob_name_suffix)
 			pick_message = "[pick_message] Auto Suffix: \"[mob_name_suffix]\" "
 		mname = sanitizeSafe(input(user, pick_message, "Name for a [species] (without prefix/suffix)"))
-	
-	if(!mname)
-		mname = pick(last_names)
+
+	if(!mname || mname == "")
```

Comment:
You can use `length(mname) == 0` instead of checking for an empty string

Reply:
```suggestion
	if(!length(mname))
```